roles/vdm/tasks/mirror.yaml

```diff
@@ -19,13 +19,30 @@
     - update
 
 - name: mirror - overlay
-  overlay_facts:
-    cadence_name: "{{ V4_CFG_CADENCE_NAME }}"
-    cadence_number: "{{ V4_CFG_CADENCE_VERSION }}"
-    existing: "{{ vdm_overlays }}"
-    add:
-      - { transformers: "mirror.yaml", vdm: true, priority: 65 }
-      - { generators: "mirror.yaml", vdm: true }
+  block:
+    - name: v4 crunchy
+      overlay_facts:
+        cadence_name: "{{ V4_CFG_CADENCE_NAME }}"
+        cadence_number: "{{ V4_CFG_CADENCE_VERSION }}"
+        existing: "{{ vdm_overlays }}"
+        add:
+          - { transformers: "mirror.yaml", vdm: true, priority: 65 }
+          - { generators: "mirror.yaml", vdm: true }
```

Comment:
Replace these:
```yaml
- { transformers: "mirror.yaml", vdm: true, priority: 65 }
- { generators: "mirror.yaml", vdm: true }
```
with:
```yaml
- { transformers: "mirror.yaml", vdm: true, priority: 65, max: 2022.09 }
- { generators: "mirror.yaml", vdm: true, max: 2022.09 }
```
Then remove the `when` statement below.

Reply:
Done.